### ISSUE TITLE
Can't build ionic 5 app with Abbyy sdk

### ISSUE THREAD

**Issue by azeroual2017** [abbyysdk/RTR-SDK.Cordova#19]
Hello,
i added abbyy sdk to my ionic 5(Angular) app, i have followed instructions mentiioned in readme file. but When i want to build andoird app in android studio, i get the following error : **path...\fti\android\capacitor-cordova-android-plugins\src\main\java\com\abbyy\mobile\rtr\javascript\text\TextRecognitionResult.java:8: error: import requires canonical name for CharInfo
import com.abbyy.mobile.rtr.IRecognitionCoreAPI.CharInfo;**

**_thanks a lot for your help_**

**image error :** 
![image](https://user-images.githubusercontent.com/25665574/123231887-eb431a80-d4d8-11eb-88ba-d75f6d04be6b.png)

**my app structure is  :** 
![tout](https://user-images.githubusercontent.com/25665574/123231302-5f30f300-d4d8-11eb-817f-a2e6d1e2fdcf.PNG)

**my android/build.gradle :**
![android-build-gradle](https://user-images.githubusercontent.com/25665574/123231451-7f60b200-d4d8-11eb-8dd8-b5e95044cbe0.PNG)

**my andorid/app/build.gradle :** 
![android-app-build-gradle](https://user-images.githubusercontent.com/25665574/123231526-92738200-d4d8-11eb-8627-49758158aa31.PNG)

**Comment by al0b** [comment]
Thank you that point us to this issue!

For hot fix you can replace two lines in TextRecognitionResult.java
```
-import com.abbyy.mobile.rtr.IRecognitionCoreAPI.CharInfo;
+import com.abbyy.mobile.rtr.TextCapture.CharInfo;
 import com.abbyy.mobile.rtr.IRecognitionCoreAPI.TextBlock;
-import com.abbyy.mobile.rtr.IRecognitionCoreAPI.TextLine;
+import com.abbyy.mobile.rtr.TextCapture.TextLine;
```

We will fix this error in the nearest relese, thank you once more time.

**Comment by al0b** [comment]
Fixed in the [1.0.17](https://github.com/abbyy/RTR-SDK.Cordova/tree/v1.0.17) .
